parser.cpp

```diff
@@ -197,17 +197,16 @@ void Parser::parseFiles(QNetworkReply *reply, QMap<QNetworkReply*, Structureelem
             filesize = fileInformation["fileSize"].toString().toInt();
             timestamp = fileInformation["modifiedTimestamp"].toInt();
             url = fileInformation["downloadUrl"].toString();
+            url = QByteArray::fromPercentEncoding(url.toLocal8Bit());
```

Comment:
Yes! Because of using QByteArray::fromPercentEncoding, my workaround is obsolete!